specification/frontdoor/resource-manager/Microsoft.Network/stable/2019-10-01/webapplicationfirewall.json

```diff
@@ -0,0 +1,812 @@
+{
```

Comment:
This file seems identical to 2019-03-01/webapplicationfirewall.json. Are there other changes expected or is this just initializing the new version?

Reply:
The first commit initializes the new version. The subsequent commits contain the proposed changes.